src/static/css/openforms.css

```diff
@@ -558,6 +558,53 @@ button.datebutton:hover {
     box-shadow: 0 0 0 2px var(--ripple-primary) !important;
 }
 
+/* Form Formatting */
```

Comment:
Can you please change this comment to something a bit more descriptive like 'Checkbox / Radio matrix' or something similar